contracts/common/EtherPaymentFallback.sol

```diff
@@ -6,11 +6,13 @@ pragma solidity >=0.7.0 <0.9.0;
 /// @author Richard Meissner - <richard@gnosis.pm>
 contract EtherPaymentFallback {
 
+    event Received(address indexed sender, uint256 value);
```

Comment:
SafeEtherReceived?

Reply:
I on purpose did not mention `Ether` or `Eth` as on ther chains this is not correct :D